pkg/products/observability/prometheusRules.go

```diff
@@ -50,16 +50,6 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 						},
 						Expr: intstr.FromString("(time() - (max( kube_job_status_start_time * ON(job_name) GROUP_RIGHT() kube_job_labels{label_monitoring_key='middleware'} ) BY (job_name, label_cronjob_name) == ON(label_cronjob_name) GROUP_LEFT() max( kube_job_status_start_time * ON(job_name) GROUP_RIGHT() kube_job_labels{label_monitoring_key='middleware'} ) BY (label_cronjob_name))) > 60*60*25"),
 					},
-					{
-						Alert: "CronJobsFailed",
```

Comment:
There may be a test update if this alert is checked for being present. but it doesn't need to block this pr.